packages/client/src/services/MetricsService.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export default class MetricsService {
+  async sendMetric(metricName: string, path: string): Promise<void> {
+    try {
+      await axios.post(`${process.env.PUBLIC_URL}/metrics`, {
+        name: `${metricName}`,
+        dimensions: {
+          url: `${path}`
+        }
+      })
+    }
+    catch (e) {
+      console.log(`unable to send metric: ${metricName} and dimension: ${path}`);
```

Comment:
Throw this error, as in this example: https://github.com/Nike-Inc/referee/blob/385bdf5b91f86214bff26f159c709b349dfab52b/packages/client/src/services/KayentaApiService.ts#L25

Reply:
also use `log.error`

Reply:
Actually, lets not crash the app over this, just log.error and swallow the error please.